guides/user/kubernetes-v2/best-practices/index.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: single
+title:  "Best Practices for the Kubernetes Provider V2"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+The Kubernetes Provider V2 enables a wide variety of ways to deploy your
+manifests into Kubernetes clusters. This page provides best-practices for doing
+so.
+
+## Deploy Docker Images by Digest
+
+If your registry exposes image changes by
+[digest](https://docs.docker.com/registry/spec/api/#content-digests), we
+recommend deploying images by their digest rather than tag, e.g.
+`gcr.io/my-image@sha256:95ff090...` rather than `gcr.io/my-image:v1`.
+
+The reasoning is: the digest is a content-addressable way of referencing
+your image, since it's derived from the hash of the image contents. If your
+manifest points to an image by its digest, you know that each time it's
+deployed, it will point to the same contents (binary, dependencies, config,
+etc...). If you rely on the tag, it's entirely possible that deploying the same
+Manifest twice can have different results.
+
+There are two ways to achieve this in Spinnaker:
+
+1. Template your manifests outside of Spinnaker to point to the digest you want
+   to deploy, before submitting them to Spinnaker.
+
+2. Rely on Spinnaker's artifact substitution and a trigger that supplies your
+   image digest.
+
+## Emit Your Deployed Manifests to an Audit Log
+
+To keep track of what's changing in your cluster, and under what circumstances
+it is changing, we highly recommend configuring Spinnaker to emit events to an
+external audit log. Spinnaker will already generate events for any events
+running through its orchestration engine, all that's needed is an endpoint to
+send them to. [This guide shows you
+how](https://blog.spinnaker.io/spinnaker-echo-google-cloud-functions-stackdriver-logging-spinnaker-audit-log-81139f084db9).
```

Comment:
You could maybe omit this sentence and instead link from "...configuring Spinnaker to emit events to an external audit log."